modules/server/src/network/resolvers/index.ts

```diff
@@ -1,8 +1,24 @@
-import { NetworkAggregationConfig } from '../types';
+import { NetworkAggregationConfig, SupportedNetworkFieldType } from '../types';
+import { createAggregationResolvers } from './aggregations';
 import { resolveRemoteConnectionNodes } from './remoteConnections';
 
-export const createResolvers = (networkConfigsWithSchemas: NetworkAggregationConfig[]) => ({
-	Query: {
-		nodes: async () => await resolveRemoteConnectionNodes(networkConfigsWithSchemas),
-	},
-});
+/**
+ * Create GQL resolvers
+ * @param networkConfigsWithSchemas
+ * @param networkFieldTypes
+ * @returns
+ */
+export const createResolvers = (
+	configs: NetworkAggregationConfig[],
+	networkFieldTypes: SupportedNetworkFieldType[],
+) => {
+	const aggregationsResolvers = createAggregationResolvers(configs, networkFieldTypes);
+
+	return {
+		Query: {
+			nodes: async () => await resolveRemoteConnectionNodes(configs),
+			aggregations: () => [], // TODO: Why do we need truthy val returned?
```

Comment:
Since we are handling the Aggregations type resolver at line 22, do we need this resolver at all? Or... why not just put our aggregationsResolvers object here instead?